lib/sources/mongodb.js

```diff
@@ -0,0 +1,54 @@
+var semver = require('semver');
+var _ = require('lodash');
+var agent = require('superagent');
+
+var SEMVER = /[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+)?/g;
+var TIMEOUT = 20000;
+var NOOP = function() {};
+
+function MongoDBSource(options) {
+  _.extend(this, {
+    name: 'mongodb',
+    url: 'https://www.mongodb.org/dl/osx',
```

Comment:
The rest of the sources are linux-based, rather than osx based (and semver.io is primarily used to provision servers), so I wonder if we should use the linux url instead? https://www.mongodb.org/dl/linux